Tests/Admin/BreadcrumbsBuilderTest.php.orig

```diff
@@ -0,0 +1,551 @@
+                'breadcrumb',
+                'link'
+            )->willReturn('create my object');
+            $menu->addChild('create my object', array(
+                'extras' => array(
+                    'translation_domain' => 'FooBundle',
+                ),
+            ))->willReturn($menu);
+        }
+        $childAdmin = $this->prophesize('Sonata\AdminBundle\Admin\AbstractAdmin');
+        $childAdmin->getTranslationDomain()->willReturn('ChildBundle');
+        $childAdmin->getLabelTranslatorStrategy()->willReturn($labelTranslatorStrategy->reveal());
+        $childAdmin->getClassnameLabel()->willReturn('my_child_class_name');
+        $childAdmin->hasRoute('list')->willReturn(false);
+        $childAdmin->getCurrentChildAdmin()->willReturn(null);
+        $childAdmin->hasSubject()->willReturn(false);
+
+        $admin->hasRoute('list')->willReturn(true);
+        $admin->isGranted('LIST')->willReturn(true);
+        $admin->generateUrl('list')->willReturn('/myadmin/list');
+        $admin->getCurrentChildAdmin()->willReturn(
+            $action == 'my_action' ? $childAdmin->reveal() : false
+        );
+        if ($action == 'list') {
+            $admin->isChild()->willReturn(true);
+            $menu->setUri(false)->shouldBeCalled();
+        }
+        $request = $this->prophesize('Symfony\Component\HttpFoundation\Request');
+        $request->get('slug')->willReturn('my-object');
+
+        $admin->getIdParameter()->willReturn('slug');
+        $admin->hasRoute('edit')->willReturn(false);
+        $admin->getRequest()->willReturn($request->reveal());
+        $admin->hasSubject()->willReturn(true);
+        $admin->getSubject()->willReturn('my subject');
+        $admin->toString('my subject')->willReturn('My subject');
+        $admin->getTranslationDomain()->willReturn('FooBundle');
+        $admin->getLabelTranslatorStrategy()->willReturn(
+            $labelTranslatorStrategy->reveal()
+        );
+        $admin->getClassnameLabel()->willReturn('my_class_name');
+
+        $menu->addChild('My class', array(
+            'uri' => '/myadmin/list',
+            'extras' => array(
+                'translation_domain' => 'FooBundle',
+            ),
+        ))->willReturn($menu->reveal());
+        $menu->addChild('My subject')->willReturn($menu);
+        $menu->addChild('My subject', array(
+            'uri' => null,
+        ))->willReturn($menu);
+        $menu->addChild('My child class', array(
+            'extras' => array(
+                'translation_domain' => 'ChildBundle',
+            ),
+            'uri' => null,
+        ))->willReturn($menu);
+        $menu->setExtra('safe_label', false)->willReturn($menu);
+        $menu->addChild('My action', array(
+            'extras' => array(
+                'translation_domain' => 'ChildBundle',
+            ),
+        ))->willReturn($menu);
+
+        $breadcrumbsBuilder->buildBreadCrumbs($admin->reveal(), $action);
+    }
+}
```

Comment:
Nope. Look closely at its name, it's a `.orig` file. This kind of file appears when you run into a git conflict, and is here in case you want to know what the file looked like before the conflict.